plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMigrateProcedure.java

```diff
@@ -84,6 +84,33 @@ public static Object[][] fileFormats()
                 .toArray(Object[][]::new);
     }
 
+    @Test(dataProvider = "fileFormats")
```

Comment:
Could you update `TestIcebergProcedureCalls` in product tests for ensuring that Spark can read the migrated tables?